Makefile

```diff
@@ -92,6 +92,7 @@ else
 GCFLAGS = -gcflags '-N -l'
 endif
 
+VERSION ?= $(shell $(GO_RUN) ./scripts/latest-version)
```

Comment:
is this used anywhere?

Reply:
Yes in the release notes script for example.